### DISCUSSION TITLE
Menu component - Hover to show in addition to click.

### DISCUSSION THREAD

**Discussion by tannguyencse19** [chakra-ui/chakra-ui#5272] [Ideas]
Add it as a prop of Menu component, i.e: `hoverToShow`. 

This will be useful when coding Navbar component. 

Just adding `useDisclosure()` hook + on mouse event. [Demo](https://codesandbox.io/s/menu-hover-94vw5). 
_However, it has bug: If you hover to `<MenuButton />`, but not hover to `<MenuList />`, then the list will keep showing until you hover to it and hover out. I think it just because I choose the wrong mouse event._

**Comment by tricia-work** [comment]
I'd love to see this as well!
